varcode/rna/principal_transcripts.py

```diff
@@ -0,0 +1,57 @@
+from .combine_genes import combine_gene_expression_levels
+
+def choose_principal_transcripts(
```

Comment:
is this mid-refactor? there's another `choose_principal_transcripts` defined above in `rna.py`. or are these different? (sorry if missing something obvious)